modules/Rock.cmake

```diff
@@ -500,6 +504,18 @@ function(rock_prepare_pkgconfig TARGET_NAME DO_INSTALL)
     set(PKGCONFIG_CFLAGS ${${TARGET_NAME}_PKGCONFIG_CFLAGS})
     set(PKGCONFIG_LIBS ${${TARGET_NAME}_PKGCONFIG_LIBS})
 
+    if (DEFINED ROCK_PUBLIC_CXX_STANDARD)
+        set(cxx_standard ${ROCK_PUBLIC_CXX_STANDARD})
+    elseif (CMAKE_VERSION VERSION_LESS "3.1")
```

Comment:
I do not see the real benefit of two ways for setting the cxx_standard. How likely to build targets with different cxx_standards in the same project in our environments?